index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap" rel="stylesheet">
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Marwan Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar">
+        <div class="name-container">
+          <a href="index.html">
+            <h2>Marwan</h2>
+          </a>
+        </div>
+        <div>
+          <div class="bar1"></div>
+          <div class="bar2"></div>
+          <div class="bar3"></div>
+        </div>
+      </header>
+    </div>
+    <div class="links-container">
+      <ul>
+        <li>
+          <a>Portfolio</a>
+        </li>
+        <li>
+          <a>About</a>
+        </li>
+        <li>
+          <a>Contact</a>
+        </li>
+        <li>
+          <a
+          ><img
+          src="assets/Normal Button/Tertiary/Disabled.png"
+          alt="Message"
+          /></a>
+        </li>
+      </ul>
+    </div>
+  </nav>
+  
+    <div class="header">
```

Comment:
- [ ] As requested by the previous reviewer, kindly use a `section` element to wrap the headline section instead of a div element.